src/main/java/com/renatusnetwork/momentum/data/stats/PlayerStats.java

```diff
@@ -1168,6 +1171,16 @@ public int getCommandSignSize() {
         return usedCommandSigns.size();
     }
 
+    public Squad getSquad() {
```

Comment:
Can we also add a inSquad() boolean function so that instead of calling getSquad() == null, we can just call that instead, that way it looks a bit cleaner?